### PR TITLE
cc13

### PR DESCRIPTION
**Pull request by MarikoAlvarado**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/multiBracketValidation/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+
+We took the approach of turning the string into an iterable array. Each iteration checks to see if the element is an open bracket and pushes open brackets into a stack and once, an element that is closing bracket is found, the top of the stack will be check to see if it matches the type of open bracket that was post previously pushed. if it is a match, pop the partner bracket from the stack and continue searching for closing brackets and their pairs from the stack. if the wrong opening bracket is at the top of the stack or if a closing bracket is found while the stack is empty, return false.
+
+## Solution
+<!-- Embedded whiteboard image -->
+
+![whiteboard](cc13.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/multiBracketValidation/README.md:17]: Looks really great what you have so far!  When you have time, come back and add code and tests then I will regrade for you!
